services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCIssuerWellKnownProvider.java

```diff
@@ -97,6 +101,17 @@ public static Map<String, SupportedCredentialConfiguration> getSupportedCredenti
                 .distinct()
                 .collect(Collectors.toMap(SupportedCredentialConfiguration::getId, sc -> sc, (sc1, sc2) -> sc1));
 
+        // Retrieving attributes from the realm
+        Map<String, SupportedCredentialConfiguration> realmAttr = fromRealmAttributes(realm.getAttributes())
+                .stream()
+                .filter(sc -> supportedFormats.contains(sc.getFormat()))
+                .distinct()
+                .collect(Collectors.toMap(SupportedCredentialConfiguration::getId, sc -> sc, (sc1, sc2) -> sc1));
```

Comment:
If multiple configuration are stored for the same format in realm attributes, only one is selected and others are discarded. It that no problem?

Reply:
Thanks for the feedback !
We're keeping the logic as-is, we do consider this behavior acceptable for our current use case.

Reply:
We can revisit it in the future.